pkgs/servers/http/bozohttpd/default.nix

```diff
@@ -0,0 +1,91 @@
+, fetchurl
+, bmake
+, groff
+, inetutils
+, wget
+, openssl
+, userSupport ? true
+, cgiSupport ? true
+, dirIndexSupport ? true
+, dynamicContentSupport ? true
+, sslSupport ? true
+, luaSupport ? true
+, lua
+, htpasswdSupport ? true
+}:
+
+let inherit (lib) optional optionals;
+in
+stdenv.mkDerivation rec {
+  pname = "bozohttpd";
+  version = "20210227";
+
+  # bozohttpd is developed in-tree in pkgsrc, canonical hashes can be found at:
+  # http://cvsweb.netbsd.org/bsdweb.cgi/pkgsrc/www/bozohttpd/distinfo
+  src = fetchurl {
+    url = "http://www.eterna.com.au/${pname}/${pname}-${version}.tar.bz2";
+    sha512 = "b838498626ffb7f7e84f31611e0e99aaa3af64bd9376e1a13ec16313c182eebfd9ea2c2d03904497239af723bf34a3d2202dac1f2d3e55f9fd076f6d45ccfa33";
+  };
+
+  # backport two unreleased commits to fix builds on non-netbsd platforms.
+  patches = [
+    # add missing `#include <stdint.h>`
+    # https://freshbsd.org/netbsd/src/commit/qMGNoXfgeieZBVRC
+    ./0001-include-stdint.h.patch
+
+    # BUFSIZ is not guaranteed to be large enough
+    # https://freshbsd.org/netbsd/src/commit/A4ueIHIp3JgjNVRC
+    ./0002-dont-use-host-BUFSIZ.patch
+  ];
+  patchFlags = [ "-p3" ];
+
+  buildInputs = [ openssl ] ++ optional (luaSupport) lua;
+  nativeBuildInputs = [ bmake groff ];
+
+  COPTS =
+    [
+      "-DNO_BLOCKLIST_SUPPORT" # unpackaged dependency: https://man.netbsd.org/blocklist.3
+    ]
+    ++ optional (!userSupport) "-DNO_USER_SUPPORT"
+    ++ optional (!dirIndexSupport) "-DNO_DIRINDEX_SUPPORT"
+    ++ optional (!dynamicContentSupport) "-DNO_DYNAMIC_CONTENT"
+    ++ optional (!luaSupport) "-DNO_LUA_SUPPORT"
+    ++ optional (!sslSupport) "-DNO_SSL_SUPPORT"
+    ++ optional (!cgiSupport) "-DNO_CGIBIN_SUPPORT"
+    ++ optional (htpasswdSupport) "-DDO_HTPASSWD";
+
+  _LDADD = [ "-lcrypt" "-lm" ]
+    ++ optional (luaSupport) "-llua"
+    ++ optionals (sslSupport) [ "-lssl" "-lcrypto" ];
+  makeFlags = [ "LDADD=$(_LDADD)" ];
+
+  doCheck = true;
+  checkInputs = [ inetutils wget ];
+  checkFlags = [ "-dx" "VERBOSE=yes" ] ++ optional (!cgiSupport) "CGITESTS=";
+
+  installPhase = ''
+    install -m755 -Dt $out/bin bozohttpd
+    install -m644 -Dt $out/share/man/man8 bozohttpd.8
+  '';
+
+  meta = with lib; {
+    description = "Bozotic HTTP server; small and secure";
+    longDescription = ''
+      bozohttpd is a small and secure HTTP version 1.1 server. Its main
+      feature is the lack of features, reducing the code size and improving
+      verifiability.
+
+      It supports CGI/1.1, HTTP/1.1, HTTP/1.0, HTTP/0.9, ~user translations,
+      virtual hosting support, as well as multiple IP-based servers on a
+      single machine. It is capable of servicing pages via the IPv6 protocol.
+      It has SSL support. It has no configuration file by design.
+    '';
+    homepage = "http://www.eterna.com.au/bozohttpd/";
+    changelog = "http://www.eterna.com.au/bozohttpd/CHANGES";
+    license = licenses.bsd2;
+    maintainers = [ maintainers.embr ];
```

Comment:
Please add yourself.

Reply:
They did.

Reply:
I may suggest renaming the name in nixpkgs or adding embr as a display to GitHub because otherwise it is not really obvious.

Reply:
People _really_ don't like it when you ask them to change their names (look how that went last time).  I think it's safe to assume in a new package with one maintainer listed, that that's the person submitting the PR.  In fact, you can tell very easily that that's the case, because OfBorg has added the "by: package mantainer" label here.

Reply:
It is my display name on github, and also my name.

Reply:
And when all else fails, you can always check the maintainer list: https://github.com/NixOS/nixpkgs/blob/f2f00af5b8e04a848c2b9ab8dcaf3e1c22a90dfb/maintainers/maintainer-list.nix#L2796-L2801

Reply:
@SuperSandro2000 it is really not your responsibility to determine if the name people are using is okay, people can use whatever display names / maintainer names / github handles they want, and they don't need to match. Please don't push back on this in the future, and let them be.

Reply:
> I think it's safe to assume in a new package with one maintainer listed, that that's the person submitting the PR.

It already happened multiple times that newcomers copied files from another package and forgot to change the maintainer field. Just making sure that isn't happening.

> In fact, you can tell very easily that that's the case, because OfBorg has added the "by: package mantainer" label here.

Good reminder. Totally forgot about that.

> It is my display name on github, and also my name.

Maybe I got trolled by GitHubs UI. Don't know anymore.